repo-sync/commands/create_pull_request.yml

```
@@ -0,0 +1,26 @@
+description: "Creates a branch, commits all changes and opens a pull request."
+
+parameters:
+  directory:
+    description: "The directory of the repository"
+    type: string
+  target_branch:
+    description: "The branch that is used as pull request target."
+    type: string
+  branch:
+    description: "The branch that the pull request is created from."
+    type: string
+  pr_label:
+    description: "The label that is applied to the pull request. Also used as branch name in combination with the current build number."
```

```suggestion
    description: "The label that is applied to the pull request."
```

From what I see the second part is not true anymore.